src/actions/recomputeReduxState.js

```diff
@@ -923,6 +923,11 @@ export const createStateFromQueryOrJSONs = ({
     measurements.collectionToDisplay = collectionToDisplay;
     controls = {...controls, ...collectionControls};
     query = updatedQuery;
+  } else {
+    // Hide measurements panel if loading failed
+    controls.panelsToDisplay = controls.panelsToDisplay.filter((panel) => panel !== "measurements");
+    controls.canTogglePanelLayout = hasMultipleGridPanels(controls.panelsToDisplay);
+    controls.panelLayout = controls.canTogglePanelLayout ? controls.panelLayout : "full";
```

Comment:
For completeness I'd also suggest removing any measurements queries. (There's no partial measurements state which is nice - it's either the loaded state or the default state.)

Reply:
Ah, good catch! Will do.

Reply:
Added in https://github.com/nextstrain/auspice/pull/1881/commits/c03d336afb0c44969b7281047f5caa8fb54154f4